wordproof-timestamp/includes/NotificationHelper.php

```diff
@@ -48,43 +48,43 @@ public function checkEditScreen()
 
   public function changePostUpdateMessages($labels)
   {
-      $labels->item_updated = __( 'Your post has been updated but is at risk. Timestamp your content now to claim ownership.' );
-      $labels->item_published = __( 'Your post is published. Timestamp your content to claim ownership.' );
+      $labels->item_updated = __( 'Your post has been updated but is at risk. Timestamp your content now to claim ownership.', 'wordproof-timestamp' );
+      $labels->item_published = __( 'Your post is published. Timestamp your content to claim ownership.', 'wordproof-timestamp' );
       return $labels;
   }
   public function changePageUpdateMessages($labels)
   {
-      $labels->item_updated = __( 'Your page has been updated but is at risk. Timestamp your content now to claim ownership.' );
-      $labels->item_published = __( 'Your page is published. Timestamp your content to claim ownership.' );
+      $labels->item_updated = __( 'Your page has been updated but is at risk. Timestamp your content now to claim ownership.', 'wordproof-timestamp' );
+      $labels->item_published = __( 'Your page is published. Timestamp your content to claim ownership.', 'wordproof-timestamp' );
       return $labels;
   }
 
   private function setupPlugin()
   {
-    $message = 'Protect your content and proof the authenticity of your content to visitors and search engines. Start the Wizard to get you started with WordProof!';
+    $message = __( 'Protect your content and proof the authenticity of your content to visitors and search engines. Start the Wizard to get you started with WordProof!', 'wordproof-timestamp' );
     $type = 'info';
-    $buttonText = 'Setup WordProof Timestamp';
+    $buttonText = __( 'Setup WordProof Timestamp', 'wordproof-timestamp' );
```

Comment:
I see `'wordproof-timestamp'` being used quite often. Maybe make a constant out of this or wrap the function `__`?